server/services/mongodb-remote/src/test/java/com/mongodb/stitch/server/services/mongodb/remote/SyncMongoClientIntTests.kt

```diff
@@ -699,114 +735,158 @@ class SyncMongoClientIntTests : BaseStitchServerIntTest() {
             assertNull(coll.findOneById(doc1Id))
 
             goOnline()
-            listenAndSync()
+            syncPass()
             assertNull(remoteColl.find(doc1Filter).first())
             assertNull(coll.findOneById(doc1Id))
-        })
+        }
     }
 
     @Test
     fun testTurnDeviceOffAndOn() {
-        testSyncInBothDirections({
+        testSyncInBothDirections {
             val coll = getTestColl()
 
             val remoteColl = getTestCollRemote()
 
             val docToInsert = Document("hello", "world")
             docToInsert["foo"] = 1
-            coll.insertOne(docToInsert)
+            remoteColl.insertOne(docToInsert)
 
-            val doc = coll.find(docToInsert).first()!!
+            val doc = remoteColl.find(docToInsert).first()!!
             val doc1Id = BsonObjectId(doc.getObjectId("_id"))
             val doc1Filter = Document("_id", doc1Id)
 
             powerCycleDevice()
-            coll.sync(doc1Id, DefaultSyncConflictResolvers.localWins())
+
+            coll.configure(DefaultSyncConflictResolvers.localWins(), null, null)
+            coll.syncOne(doc1Id)
             powerCycleDevice()
-            listenAndSync() // does nothing
 
-            coll.sync(doc1Id, DefaultSyncConflictResolvers.localWins())
-            listenAndSync() // syncs this time
+            coll.configure(DefaultSyncConflictResolvers.localWins(), null, null)
```

Comment:
Based on the diff here, the comment should be that it syncs this time right? The second syncone and syncpass shouldn't be necessary